modules/fedimint-dummy-tests/tests/tests.rs

```diff
@@ -70,3 +78,29 @@ async fn federation_should_abort_if_balance_sheet_is_negative() -> anyhow::Resul
 
     Ok(())
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn unbalanced_transactions_get_rejected() -> anyhow::Result<()> {
+    let fed = fixtures().new_fed().await;
+    let client = fed.new_client().await;
+
+    let amount = sats(1000);
```

Comment:
I'm only nitpicky because I'm trying to understand what is this test doing.

This `amount` is kind of vague. Would inline and rename to `output_value` or something.